src/rt/typeinfo/ti_double.d

```diff
@@ -67,12 +67,6 @@ class TypeInfo_d : TypeInfo
 
     override @property immutable(void)* rtInfo() nothrow pure const @safe { return rtinfoNoPointers; }
 
-    version (Windows)
-    {
-    }
```

Comment:
Why is the special case for Windows no longer necessary?

Reply:
It never was, as floats and doubles are passed in a XMM on Win64 too. On 32-bit x86, XMMs aren't used at all.